### PR TITLE
Improve CLS log messages

### PR DESCRIPTION
**Pull request by suleymanakbas91**
**Description**

Changes proposed in this pull request:

- Improve CLS log messages

### CODE REVIEW

**Review by skhalash — DISMISSED** [review]
Good job!

**Review by skhalash — APPROVED** [review]

File: components/kyma-environment-broker/internal/process/provisioning/cls_binding_step.go [hunk 1]
```diff
@@ -124,8 +124,8 @@ func (s *ClsBindStep) Run(operation internal.ProvisioningOperation, log logrus.F
 		// fetch existing overrides
 		overrideParams, err = cls.DecryptOverrides(s.secretKey, operation.Cls.Overrides)
 		if err != nil {
-			failureReason := fmt.Sprintf("decryptClsOverrides() call failed")
-			log.Errorf("%s: %s", failureReason, err)
+			failureReason := fmt.Sprintf("Unable to decrypt CLS overrides")
```
Comment (polskikiel) [inline components/kyma-environment-broker/internal/process/provisioning/cls_binding_step.go:127]: we don't need to use `Sprintf()`. Please fix for other occurrences without parameters
```suggestion
			failureReason := "Unable to decrypt CLS overrides"
```

Comment (suleymanakbas91) [reply components/kyma-environment-broker/internal/process/provisioning/cls_binding_step.go]: Fixed

**Review by polskikiel — APPROVED** [review]
lgtm

**Pull request merged**